env0/data_organization_test.go

```diff
@@ -0,0 +1,36 @@
+package env0
+
+import (
+	"fmt"
+	"github.com/env0/terraform-provider-env0/client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"testing"
+)
+
+var resourceType = "env0_organization"
+var resourceName = "test"
+var resourceFullName = fmt.Sprintf("%s.%s", resourceType, resourceName)
+var organization = client.Organization{
+	Id:   "id0",
+	Name: "name0",
+}
```

Comment:
@razbensimon so I wouldn't forget or someone will miss it - please avoid using global values. These are global for the package scope 😨